elements/rh-alert/README.md

```diff
@@ -78,3 +78,15 @@ import '@rhds/elements/rh-alert/rh-alert.js';
     <button slot="actions" data-action="confirm">Confirm</button>
 </rh-alert>
 ```
+
+### Adding an Event Listener to a Basic Alert
+
+If you would like to add custom logic when the alert is closed, you can do so with JavaScript. 
+```js
+  // Query for the alert element
+  const alert = document.querySelector('rh-alert');
+  alert.addEventListener('close', () => {
+    // Add code to be executed when the alert element is closed.
+  });
```

Comment:
```suggestion
  alert.addEventListener('close', async function(event) {
    // Prevent the alert from closing
    event.preventDefault();
    // Perform some async task
    await alertClosed();
    // Close the alert when finished
    this.remove();
  });
```
I'm not sure why the alert's private `#closeHandler` sets the state to `'error'`, that might interfere with certain workflows. Perhaps we should remove that line?

Reply:
Added this in as another example